dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -60,10 +60,13 @@ recursiveSearch=Perform a nested group search
 recursiveSearch.desc=Performs a nested group search. Select this option only if the LDAP server does not support recursive server-side searches.
 
 searchTimeout=Search timeout
-searchTimeout.desc=Maximum time for an LDAP server to respond before a request is canceled. This is equivalent to a read timeout once the connection is established.
+searchTimeout.desc=Maximum time for an LDAP server to respond before a request is canceled.
 
-connectTimeout=Timeout duration for establishing LDAP connection
-connectTimeout.desc=Maximum time for establishing a connection to the LDAP server. The program logs an error message if the specified time expires.
+connectTimeout=Timeout for establishing LDAP connection
+connectTimeout.desc=Maximum time for establishing a connection to the LDAP server. A value of 0 indicates that the TCP protocol's timeout value should be used. The program logs an error message if the specified time expires.
+
+readTimeout=Timeout for reading from an LDAP connection
+readTimeout.desc=Maximum time to wait for read operations for LDAP operations. A value of 0 indicates that there is no timeout and the read will wait indefinitely. The program logs an error message if the specified time expires.
```

Comment:
Change "A value of 0 indicates that there is no timeout and the read will wait indefinitely." to "A value of 0 indicates that no timeout exists and that the read waits indefinitely."

Reply:
I have updated the message as suggested.